src/_00_Dynamic_Integer_Array/DynamicIntegerArray.java

```diff
@@ -1,95 +1,124 @@
+		// element longer than the member array
+		int[] temp = new int[num.length+1];
+		// B. Make a for loop that iterates through the new array
+		for (int i = 0; i < temp.length; i++) {
+			if(i<location) {
+				temp[i] = num[i];
+			}
+			if(i>location) {
+				temp[i] = num[i-1];
+			}
+			if(i==location) {
+				temp[i] = v;
+			}
 			
-			//E. else, set the element at i of the new array to the value v
-		
-		
-		//F. set the member array equal to the new array
-	
+		}
+		num = temp;
+		// C. if i is less than location:
+		// set the element at i of the new array to the element at i of the member array
+
+		// D. else if i is greater than location:
+		// set the element at i of the new array to the i - 1 element of the member
+		// array
+
+		// E. else, set the element at i of the new array to the value v
+
+		// F. set the member array equal to the new array
+
 	}
-	
-	//8. Run the tests again and check your progress
-	
-	//9. Complete the steps in the remove method
+
+	// 8. Run the tests again and check your progress
+
+	// 9. Complete the steps in the remove method
 	public void remove(int location) {
-		//A. create a new array that is one element smaller than the member array
+		// A. create a new array that is one element smaller than the member array
+		int[] temp = new int[num.length-1];
+		// B. make a for loop to iterate through the member array
+		for (int i = 0; i < num.length; i++) {
+			if(i<location) {
+				temp[i] = num[i];
+			}
+			if(i>location) {
+				temp[i-1] = num[i];
+			}
 		
-		//B. make a for loop to iterate through the member array
-		
-			//C. if i  is less than location
-			//		set the element at i of the new array to the element at i of the member array
-			
-			//D. if i  is greater than location
-			//		set the element at i - 1 of the new array to the element at i of the member array
-			
-			//E. else, continue;
-			
-		//F. set the member array equal to the new array
+		}
+		num = temp;
+		// C. if i is less than location
+		// set the element at i of the new array to the element at i of the member array
+
+		// D. if i is greater than location
+		// set the element at i - 1 of the new array to the element at i of the member
+		// array
+
+		// E. else, continue;
+
+		// F. set the member array equal to the new array
 	}
-	
-	//10. Run the tests again the see if you are correct so far
-	
-	//11. Complete the size method so that it returns the length of the member array.
+
+	// 10. Run the tests again the see if you are correct so far
+
+	// 11. Complete the size method so that it returns the length of the member
+	// array.
 	public int size() {
-		return 0;
+		return num.length;
 	}
-	
-	//12. Complete the clear array so that it sets the member array 
-	//    equal to a new integer array of size 0
+
+	// 12. Complete the clear array so that it sets the member array
+	// equal to a new integer array of size 0
 	public void clear() {
+		int[] temp = new int[0];
+		num = temp;
```

Comment:
You can instantiate the num to be an empty array instead of using the temp variable